sdk/common/build.gradle

```diff
@@ -96,12 +96,13 @@ dependencies {
     testCompile "org.awaitility:awaitility:${awaitilityVer}"
 }
 
-check.finalizedBy jacocoTestReport
-
-jacocoTestReport {
+// squelch code coverage report for this project
+// check.finalizedBy jacocoTestReport
+/*jacocoTestReport {
     reports {
         xml.enabled false
         csv.enabled false
         html.destination "${buildDir}/jacocoHtml"
     }
 }
+*/
```

Comment:
Are we going to re-enable this, later?

Reply:
I considered either removing the coverage report configuration or commenting it out. I intentionally chose to clearly indicate that we have turned off the coverage report for this project. This is to stave off rework when someone re-enables the coverage report due to being unaware of the team's (contributor set's) consensus opinion that the coverage for this specific project should be squelched. Otherwise said, yes, project knowledge could be gleaned from other sources, but in this case the likelihood that the knowledge would be gleaned is low and the cost of repeating this work is high enough to merit IMO commenting here.

Reply:
Nick has a JIRA to see if it's worth adding test coverage to `common`. If it is, he'll reenable this.

Reply:
cc @nickbp